### PR TITLE
Translating Missing Information on French Neos 

### PR DESCRIPTION
**Pull request by Xqua**
There was a lot of work needed to get the FR locale up to par with the current Neos. 

This is now done at 100% matching the en-us local. 

This was done using BabelEdit.

### CODE REVIEW

**Comment by Frooxius** [comment]
Thanks for updating the translation! Is it possible to preserve the formatting of the original document though please? Changing the formatting makes the diff pretty much useless, so I can't really track what has actually changed in the document.

**Comment by Xqua** [comment]
@Frooxius That should do it ;) 

Feel free to cherry pick the Cleaner or not, and the added README lines.

**Comment by Frooxius** [comment]
That fixed it, thank you and thanks for the script!

File: fr.json [hunk 1]
```diff
@@ -95,6 +111,7 @@
         "World.Error.SecurityViolation" : "Violation de sécurité",
         "World.Error.OnlyRegisteredUsers" : "Seul les utilisateurs enregistrés peuvent joindre cette session",
         "World.Error.OnlyContacts" : "Seul les amis de l'hôte peuvent joindre cette session",
+        "World.Error.OnlyContactsOfContacts" : "Seul les amis des utilisateurs dans la session sont autoriser a rejoindre",
```
Comment (samuellapointe) [inline fr.json:114]: sont autorisés

File: fr.json [hunk 2]
```diff
@@ -349,6 +386,7 @@
         "Notifications.TransferedCredits" : "A transféré {amount} {token}",
         "Notifications.ReceivedTip" : "Pourboire reçu: {amount} {token}",
         "Notifications.SentTip" : "Pourboire envoyé: {amount} {token}",
+        "Notifications.VoiceMessage" : "Envoyé un message vocale",
```
Comment (samuellapointe) [inline fr.json:389]: Envoyer un message vocal

File: fr.json [hunk 3]
```diff
@@ -791,6 +841,7 @@
         "Settings.SpeedExponent" : "Exposant de la vitesse",
         "Settings.MoveThreshold" : "Zone morte de mouvements",
         "Settings.ControllerVibration" : "Vibration des contrôleurs",
+        "Settings.Haptics" : "Retour haptiques",
```
Comment (samuellapointe) [inline fr.json:844]: retours ou bien haptique, un est au singulier et l'autre au pluriel

File: fr.json [hunk 4]
```diff
@@ -951,22 +1016,228 @@
...
+        "Inspector.DynamicBoneChain.AlwaysGrabLastBone" : "Toujours attraper le dernier os",
+
+        "Wizard.General.ProcessRoot" : "Racine du Processus:",
+        "Wizard.General.Result" : "Résultat:",
+        "Wizard.General.ErrorNoRoot" : "Pas de Racine sélectionné",
+        "Wizard.General.RemovedResult" : "{n,plural, one {# Enlevé} other {# Enlevés}}",
+
+        "Wizard.AssetOptimization.Title" : "Assistant d'optimisation des objets",
+        "Wizard.AssetOptimization.CleanupUnusedAssets" : "Nettoyer les objets non utilisé",
+        "Wizard.AssetOptimization.CleanupEmptySlots" : "Netoyer les objets Vides",
```
Comment (samuellapointe) [inline fr.json:1055]: Nettoyer

File: fr.json [hunk 5]
```diff
@@ -951,22 +1016,228 @@
...
+        "Wizard.ReflectionProbes.ProcessDisabled" : "Processus désactivé",
+        "Wizard.ReflectionProbes.WithTag" : "Avec Étiquette:",
+        "Wizard.ReflectionProbes.TeleportUserToProbe" : "Téléporte moi a chaque sonde",
+        "Wizard.ReflectionProbes.DelayBetweenProbes" : "Délais entre les sondes",
+        "Wizard.ReflectionProbes.ShowDebugVisuals" : "Montrer les Visuels de Débogage",
+        "Wizard.ReflectionProbes.HideDebugVisuals" : "Cacher les Visuels de Débogage",
+        "Wizard.ReflectionProbes.BakeProbes" : "Pre-Calculer les Sondes",
+        "Wizard.ReflectionProbes.Baking" : "Pre-calcul {index} sur {count}...",
+
+        "Tutorial.Welcome.WelcomeTo" : "Bienvenue a",
```
Comment (samuellapointe) [inline fr.json:1103]: Bienvenue à

File: fr.json [hunk 6]
```diff
@@ -951,22 +1016,228 @@
...
+        "Tutorial.Welcome.LetsStart" : "C'est partie!",
```
Comment (samuellapointe) [inline fr.json:1104]: parti

File: fr.json [hunk 7]
```diff
@@ -951,22 +1016,228 @@
...
+        "Tutorial.Welcome.ScreenWarning" : "<color=#f00><b>WARNING:</b> </color> Vous jouez actuellement en mode écran, qui n'est actuellement pas officiellement pris en charge et est principalement conçu pour le débogage. <br> <br> Bien qu'il puisse être utilisé pour jouer, certaines choses ne fonctionneront pas et les commandes peuvent être difficiles à utiliser. <br> <br> <b> Neos est actuellement conçu pour être joué en RV. </b> <br> <br> Un mode bureau fonctionnel devrait être implémenté à l’avenir.",
+        "Tutorial.Welcome.ScreenUnderstand" : "Je comprend",
```
Comment (samuellapointe) [inline fr.json:1106]: comprends

File: fr.json [hunk 8]
```diff
@@ -951,22 +1016,228 @@
...
+        "Tutorial.Welcome.SettingsHeight" : "Votre taille",
+        "Tutorial.Welcome.SettingsHeightExamples" : "Exemples: 1.85 m, 185 cm, 6' 1\", 6 feet 1 inch",
+
+        "Tutorial.Welcome.StartTutorialHeader" : "Commencer le Tutoriel",
+        "Tutorial.Welcome.StartTutorialDescription" : "Nous vous transférerons dans un monde de tutoriels pour vous aider à démarrer avec Neos. Vous pouvez choisir de rendre public le monde du tutoriel, permettant à la communauté Neos de se joindre et de vous aider, ou d'apprendre par vous-même.",
+        "Tutorial.Welcome.StartTutorialPublic" : "Je veux que les gens puissent me rejoindre et m'aider",
+        "Tutorial.Welcome.StartTutorialPrivate" : "Je veux apprendre seul.",
+        "Tutorial.Welcome.PrivateTutorialHeader" : "Commencer vraiment en privé?",
+        "Tutorial.Welcome.PrivateTutorialDescription" : "Les membres de la communauté peuvent vous aider à vous installer plus facilement et plus rapidement, vraiment procéder seuls?",
+        "Tutorial.Welcome.PrivateTutorialConfirm" : "Créer en Privée",
```
Comment (samuellapointe) [inline fr.json:1138]: privé

File: fr.json [hunk 9]
```diff
@@ -951,22 +1016,228 @@
...
+        "Tutorial.Tooltip.GeneralSecondary" : "Action secondaire",
+        "Tutorial.Tooltip.ContextMenu" : "Accéder au menu d'outils et à l'option de deséquipement",
+        "Tutorial.Tooltip.DevPrimary" : "Interagir avec les gizmos (maintenir)",
+        "Tutorial.Tooltip.DevSecondary" : "Sélectionner (Montre les gizmos)",
+        "Tutorial.Tooltip.Duplicator" : "Dupliquer (En maintenant un objet)",
+        "Tutorial.Tooltip.GluePrimary" : "Appliquer la colle (maintenir)",
+        "Tutorial.Tooltip.GlueSecondary" : "Forcer le séchage de la colle",
+        "Tutorial.Tooltip.ColorPrimary" : "Appliquer la Couleur",
+        "Tutorial.Tooltip.ColorSecondary" : "Sélectionner la Couleur",
+        "Tutorial.Tooltip.Brush" : "Déssiner",
```
Comment (samuellapointe) [inline fr.json:1218]: dessiner

File: fr.json [hunk 10]
```diff
@@ -951,22 +1016,228 @@
...
+        "Tutorial.Tooltip.MeshSecondary" : "Extraire le maillage",
+        "Tutorial.Tooltip.CharacterColliderPrimary" : "Ajouter un collisioneur de personnage",
+        "Tutorial.Tooltip.CharacterColliderSecondary" : "Enlever un collisioneur de personnage",
+        "Tutorial.Tooltip.GrabbableSetterPrimary" : "Appliquer Aggripable",
+        "Tutorial.Tooltip.GrabbableSetterSecondary" : "Enlever Aggripable",
+        "Tutorial.Tooltip.LightPrimary" : "Créer une lumière (maintenir)",
+        "Tutorial.Tooltip.LightSecondary" : "Changer la couleur",
+        "Tutorial.Tooltip.MeshVisibilityPrimary" : "Rendre Visible",
+        "Tutorial.Tooltip.MeshVisibilitySecondary" : "Rendre Invisible",
+        "Tutorial.Tooltip.ShapePrimary" : "Creer une forme",
```
Comment (samuellapointe) [inline fr.json:1231]: Créer

**Comment by samuellapointe** [comment]
Added some late comments for grammar

**Comment by Xqua** [comment]
Thanks @samuellapointe ! Great catch :)

**Pull request merged**